l10n_it_reverse_charge/tests/test_rc.py

```diff
@@ -171,6 +171,55 @@ def test_supplier_extraEU_no_outstanding_payment(self):
         payments_lines = (self_purchase_payment | self_purchase_rc_payment).line_ids
         self.assertTrue(all(payments_lines.mapped("reconciled")))
 
+    def test_supplier_extraEU_no_outstanding_payment_different_currencies(self):
+        """
+        Self invoice from Extra EU partner in a different currency
+        """
+        invoice = self.create_invoice(
+            self.supplier_extraEU_EUR,
+            amounts=[100],
+            taxes=self.tax_22ae,
+            currency=self.env.ref("base.EUR"),
+        )
+
+        self_invoice = invoice.rc_self_invoice_id
+        self_payment = self_invoice.rc_payment_move_id
+        # check self payment creation
+        self.assertTrue(self_payment)
+        # check self payment amount total
+        self.assertEqual(self_payment.amount_total, 144.0)
+        # check self payment amount total in currency
+        self.assertEqual(self_payment.amount_total_signed, 130.91)
+        # check self payment amount net pay residual
+        self.assertEqual(self_payment.amount_net_pay_residual, -77.82000000000001)
```

Comment:
Ho provato ad eseguire il test in un ambiente dove ho solo installato `l10n_it_reverse_charge` e ottengo:
```
ERROR: TestReverseCharge.test_supplier_extraEU_no_outstanding_payment_different_currencies
Traceback (most recent call last):
  File "/path/to/l10n-italy/l10n_it_reverse_charge/tests/test_rc.py", line 194, in test_supplier_extraEU_no_outstanding_payment_different_currencies
    self.assertEqual(self_payment.amount_net_pay_residual, -77.82000000000001)
AttributeError: 'account.move' object has no attribute 'amount_net_pay_residual'
```

Perché il campo `amount_net_pay_residual` è definito nel modulo `l10n_it_withholding_tax`
https://github.com/OCA/l10n-italy/blob/c3a0791cd30a9252133b31574e3a87511c8e02cd/l10n_it_withholding_tax/models/account.py#L335

che non è una dipendenza di `l10n_it_reverse_charge`, si può rimuovere questo `assert`?

Reply:
fatto

Reply:
:+1: